component/src/main/java/com/celements/structEditor/fields/SelectTagAutocompletePageType.java

```diff
@@ -73,6 +74,11 @@ public void collectAttributes(AttributeBuilder attrBuilder, DocumentReference ce
       if ((separator.isPresent())) {
         attrBuilder.addNonEmptyAttribute("data-separator", separator.get());
       }
+      String docFN = getStructDataEditorService().getCellValueAsString(cellDocRef,
+          modelContext.getDoc()).or("");
+      if (!Strings.isNullOrEmpty(docFN)) {
```

Comment:
instead of a `isNullOrEmpty` you could simply check the returned `Optional#isPresent`, that's what it is for